yala/linters.py

```diff
@@ -173,5 +173,30 @@ def parse(self, stdout_lines, stderr_lines):
         return self._parse_by_pattern(stdout_lines, pattern), stderr_lines
 
 
+class Black(Linter):
+    """Parser for black code formatter lint check."""
+
+    name = 'black'
+    command = 'black --check'
+
+    def parse(self, stdout_lines, stderr_lines):
+        """Parse linter stdout and stderr lines.
+
+        Expected error message:
+
+        would reformat file1.py
+        would reformat file2.py
+        Oh no! 💥 💔 💥
+        2 files would be reformatted.
+        """
+        pattern = re.compile(r'''^.*?would\sreformat\s(?P<path>.+\.py)$''',
+                             re.VERBOSE | re.IGNORECASE)
```

Comment:
Is it possible to do it without `re.IGNORECASE`?

Reply:
Yep. Done here 0c95924c99c70b9b5e4e3141d9dda83a699d3888